libbpf-tools/trace_helpers.c

```diff
@@ -395,3 +398,112 @@ bool is_kernel_module(const char *name)
 	fclose(f);
 	return found;
 }
+
+bool fentry_exists(const char *name, const char *mod)
+{
+	const char sysfs_vmlinux[] = "/sys/kernel/btf/vmlinux";
+	struct btf *base, *btf = NULL;
+	const struct btf_type *type;
+	const struct btf_enum *e;
+	char sysfs_mod[64];
```

Comment:
max module name length is 56 , so this should be 16 + 1 + 56 = 73 to accomodate any possible module name, let's make it 80 for the round value?